src/components/rich-text-editor/rich-text-editor.ts

```diff
@@ -81,6 +82,22 @@ export class MRichTextEditor extends ModulVue implements InputManagementData, In
     @Prop()
     public scrollableContainer: string | undefined;
 
+    @Prop({
+        default: 5,
+        validator: (level: number) => {
+            return level >= 1 && level <= 6;
+        }
+    })
+    public firstHeaderLevel: number;
+
+    @Prop({
+        default: 6,
+        validator: (level: number) => {
+            return level >= 1 && level <= 6;
+        }
+    })
+    public lastHeaderLevel: number;
```

Comment:
Je ne sais pas si ce serait possible de valider que lastHeaderLevel est plus grand ou égal à firstHeaderLevel...

Reply:
Ce n'est pas possible de valider une prop contre l'autre
=> https://github.com/vuejs/vue/issues/3495

AU PIRE on pourrait se faire un watch
@watch('frstHeaderLevel')
@watch('lastHeaderLevel')
validateHeaderLevel(): void {
    // Valider ici
}

Pis sur le created faut aussi caller this.validerHeaderLevel

Sinon je ferais aussi un 

function validerHeaderBound(value: number) {
    return value>= 1 && value<= 6;
}

Pis je le callerais dans chacun des validateurs.

Reply:
Bonne idée!

Reply:
La première partie est faite. La 2e ne peut être faite comme tu le veux: Notez que les props sont validées avant que l’instance du composant soit créée. Donc les propriétés d’instances (par ex. data, computed, etc) ne seront pas disponibles dans les fonctions default ou validator.

Pour l'instant j'ai laissé les validations dans le validator.